src/browser/api/window.js

```diff
@@ -2409,6 +2412,13 @@ function restoreWindowPosition(identity, cb) {
                 break;
         }
 
+        // set zoom level
+        let level = savedBounds.zoomLevel;
+        if (level) {
```

Comment:
What about the case where `level` is 0?

Reply:
That's default value. So no need to call the function.

Reply:
Oh yea you're right. Windows by default start at 0.